search/src/test/scala/weco/api/search/services/AggregationsTest.scala

```diff
@@ -148,35 +148,19 @@ class AggregationsTest
             List(WorkAggregationRequest.Format, WorkAggregationRequest.Subject),
           filters = List(
             FormatFilter(List("a")),
-            SubjectLabelFilter(Seq("pGkJTZWwn4"))
```

Comment:
This subject does not appear to exist in the test dataset. I'm a bit surprised that this test has been passing.  I suspect it's something to do with returning 0-count buckets.